app/presenters/school_vacancies_presenter.rb

```diff
@@ -13,18 +15,27 @@ def self.valid_types
   end
 
   def draft
-    @school.vacancies.draft
+    sort_vacancies(@school.vacancies.draft)
   end
 
   def pending
-    @school.vacancies.pending
+    sort_vacancies(@school.vacancies.pending)
   end
 
   def expired
-    @school.vacancies.expired
+    sort_vacancies(@school.vacancies.expired)
   end
 
   def published
-    @school.vacancies.live
+    sort_vacancies(@school.vacancies.live)
+  end
+
+  private
+
+  def sort_vacancies(vacancies)
+    vacancies = vacancies.sort_by { |vacancy| vacancy[sort.column] }
```

Comment:
You're probably better off doing the sorting at a database level using [order](https://apidock.com/rails/ActiveRecord/QueryMethods/order) than loading doing a sort at the code level

Reply:
Done.